src/engraving/libmscore/ornament.cpp

```diff
@@ -229,6 +243,11 @@ void Ornament::computeNotesAboveAndBelow(AccidentalState* accState)
         }
 
         Note*& note = _notesAboveAndBelow.at(i);
+        if (!note && above && _cueNoteChord) {
+            assert(_cueNoteChord->notes().size());

```

Comment:
There is already the same assert inside Chord::upNote()

Reply:
ha! thanks 👍